spring-boot-starter-mesos/src/test/java/com/containersolutions/mesos/scheduler/requirements/ports/PortUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.containersolutions.mesos.scheduler.requirements.ports;
+
+import org.apache.mesos.Protos;
+
+import static com.containersolutions.mesos.config.autoconfigure.MesosSchedulerConfiguration.MESOS_PORTS;
+
+public class PortUtil {
```

Comment:
There's already a `com.containersolutions.mesos.TestHelper` class, where these small helper method will fit in very nicely.